lldb/source/Plugins/Platform/Android/AdbClientUtils.cpp

```diff
@@ -0,0 +1,162 @@
+//===-- AdbClientUtils.cpp ------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#include "lldb/Utility/Connection.h"
+#include "AdbClientUtils.h"
+#include "lldb/Utility/LLDBLog.h"
+#include "lldb/Utility/Log.h"
+#include "lldb/Utility/Status.h"
+#include "lldb/Utility/Timeout.h"
+#include <chrono>
+#include <cstdlib>
+#include <sstream>
+
+using namespace lldb;
+using namespace lldb_private;
+using namespace lldb_private::platform_android;
+using namespace std::chrono;
+
```

Comment:
Usually, "using namespace" should be put under a smaller scope instead of global. Consider moving these using statements to line 26.